### PR TITLE
Adds custom analyzer to index using s_stemmer

### PR DESCRIPTION
**Pull request by kenoir**
See https://github.com/guardian/grid/issues/819

### CODE REVIEW

**Comment by theefer** [comment]
:+1: LGTM, scary part will be the reindex

**Comment by jamesgorrie** [comment]
RE: reindexing - We should potentially (if it doesn't add overhead) be looking at our [own tool](https://github.com/guardian/elasticsearch-remap-tool) for that.

**Comment by kenoir** [comment]
Going to merge this for now as it has no impact apart from when creating a new index.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Settings.scala [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  )
+
+  val filter = Json.obj(
+    "s_stemmer" -> Json.obj(
+      "type" -> "stemmer",
+      "language" -> "minimal_english"
+    ),
+    "gu_stopwords" -> Json.obj(
+      "type" -> "stop",
+      "stopwords" -> "_english_"
```
Comment (akash1810) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Settings.scala]: Might be missing something, but is this the standard list of stopwords?

Comment (kenoir) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Settings.scala]: It is, you might be able to get away with not defining tit.

**Pull request merged**